app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CommentsController < ApplicationController
+  def new
+    comment = Comment.new
+    respond_to do |format|
+      format.html { render :new, locals: { comment: } }
+    end
+  end
+
```

Comment:
Great job @CindyDorantes 🤝

- [ ] [Optional] You may also add a params method to the comments_controller. 
![params-comments](https://user-images.githubusercontent.com/63917725/215248737-2b497355-b05a-41b8-bacd-bba4e36512a9.PNG)
You can learn more about strong parameters by following this [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).

Reply:
Of course! Thanks